doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -86,7 +77,7 @@ Matter
   * Default support for nRF7002 revision B.
   * Specific QR code and onboarding information in the documentation for each :ref:`Matter sample <matter_samples>` and the :ref:`Matter weather station <matter_weather_station_app>`.
   * The Bluetooth LE advertising arbiter class that enables easier coexistence of application components that want to advertise their Bluetooth LE services.
-  * Support for erasing settings partition during DFU over Bluetooth LE SMP for the Nordic nRF52 Series' SoCs.
+  * Support for erasing settings partition during DFU over Bluetooth LE SMP for the Nordic Semiconductor's nRF52 Series' SoCs.
```

Comment:
```suggestion
  * Support for erasing settings partition during DFU over Bluetooth LE SMP for the Nordic nRF52 Series' SoCs.
```
The original text matches requirements from Marketing: https://projecttools.nordicsemi.no/confluence/display/MAR/Product+names